packages/suite/src/views/settings/device/FirmwareVersion.tsx

```diff
@@ -69,6 +69,16 @@ export const FirmwareVersion = ({ isDeviceLocked }: FirmwareVersionProps) => {
     const currentFwVersion = getFwVersion(device);
     const availableFwVersion = getFwUpdateVersion(device);
     const { revision } = device.features;
+    const githubUrl = GITHUB_FW_COMMIT_URL + revision;
+    const githubButtonIcon = revision ? 'EXTERNAL_LINK' : undefined;
+
+    const handleAction = () => goto('firmware-index', { params: { cancelable: true } });
+
+    const GithubButton = () => (
```

Comment:
Why is this needed? 🤔

Reply:
The `revision` is `undefined` in bootloader mode, so the link does not work. Therefore, I remove the link wrapper when `revision` is not available and assign the button to a variable to avoid duplicate code.